integreat_cms/cms/forms/regions/region_form.py

```diff
@@ -832,6 +849,7 @@ def duplicate_pages(
                 row_logging_prefix,
                 keep_status,
                 offers_to_discard,
+                only_root,
             )
 
```

Comment:
To me, 100 lines are a bit too long for one method. Do you think there is a quick way to extract some parts of this method into its own method - or do we instead need a refactoring ticket?

Reply:
Neither, I think. Of the 105 lines, 10 are the function signature, 15 are just calls to the logger, and 31 are comments. (If anything, I would like to remove the in-line coments, they are pretty clear examples of superfluous comments stating WHAT is going on, instead of WHY, despite it being obvious.)

The actual logic of the function is really well written IMO (not talking about my changes, just the "original function" 😄).
The only thing that can really be extracted - the copying of the translations, in addition to the page itself - is already a separate function.

Reply:
>  (If anything, I would like to remove the in-line coments, they are pretty clear examples of superfluous comments stating WHAT is going on, instead of WHY, despite it being obvious.)

In my eyes you can do so :)

> The only thing that can really be extracted - the copying of the translations, in addition to the page itself - is already a separate function.

Alright. I just wanted to have some more time to think about it, but I think you're right :) I think we can merge this PR :)